alpakaConfig.cmake

```diff
@@ -628,7 +630,7 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 ENDIF()
                 # avoids warnings on host-device signatured, default constructors/destructors
                 IF(CUDA_VERSION GREATER_EQUAL 9.0)
-                    LIST(APPEND CUDA_NVCC_FLAGS "-Xcudafe --diag_suppress=esa_on_defaulted_function_ignored")
+                    LIST(APPEND CUDA_NVCC_FLAGS "-Xcudafe" "--diag_suppress=esa_on_defaulted_function_ignored")
```

Comment:
should work because of the single flags, but maybe without quotes are less error-prone in the future (it is intriguing to just add another flag and then again we have the issue. This cannot happen when you do not use quotes for cmake list argument.